docs: add a warning about deleting registered Edge hosts (#3626)
spectrocloud/librarium: docs/docs-content/clusters/edge/site-deployment/site-installation/edge-host-registration.md

You have the following options to register the Edge host with Palette.

:::warning

> If you are registering an Edge host that was previously registered with Palette after a reinstall, ensure that you

Reviewer: ```suggestion
If you using an Edge host that was previously registered with Palette, ensure that you
```

Reviewer: I feel like the work register was repeating too much here. 🙈
